R/foreign-keys.R

```diff
@@ -568,6 +568,21 @@ check_fk <- function(t1, t1_name, colname, t2, t2_name, pk) {
   )
 }
 
+fk_table_to_def_fks <- function(table,
+                                child_table = "child_table",
+                                child_fk_cols = "child_fk_cols",
+                                parent_table = "parent_table",
+                                parent_key_cols = "parent_key_cols") {
+  group_by(table, !!ensym(parent_table)) %>%
```

Comment:
When using the pipe, the LHS should be naked and not a function call. It's easier to read and also makes it easier to insert something between `table` and `group_by()` .

```suggestion
  table %>%
    group_by(!!ensym(parent_table)) %>%
```